examples/Example/src/prot_reanimated/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { useEffect } from 'react';
+import { useState } from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import { GestureMonitor, Gesture } from 'react-native-gesture-handler';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withSpring,
+} from 'react-native-reanimated';
+import { useAnimatedGesture } from '../useAnimatedGesture';
+import { useRef } from 'react';
+
+function Draggable() {
+  const [a, sa] = useState(1);
+
+  useEffect(() => {
+    setInterval(() => {
+      sa((a) => a + 1);
+    }, 1000);
+  }, []);
+
+  const pressed = useSharedValue(false);
+  const offsetX = useSharedValue(0);
+  const offsetY = useSharedValue(0);
+  const startX = useSharedValue(0);
+  const startY = useSharedValue(0);
+
+  const ref = useRef();
+
+  const animatedStyles = useAnimatedStyle(() => {
+    return {
+      transform: [
+        { translateX: offsetX.value },
+        { translateY: offsetY.value },
+        { scale: withSpring(pressed.value ? 1.2 : 1) },
```

Comment:
You can use one object to group those.

Reply:
Can you be more specific? Because
```js
{ 
  translateX: offset.value.x,
  translateY: offset.value.y,
  scale: withSpring(pressed.value ? 1.2 : 1),
}
```
throws an error: `You must specify exactly one property per transform object.`.